app/controllers/nonsipp/employercontributions/EmployerContributionsMemberListController.scala

```diff
@@ -63,10 +62,14 @@ class EmployerContributionsMemberListController @Inject()(
 
   def onPageLoad(srn: Srn, page: Int, mode: Mode): Action[AnyContent] = identifyAndRequireData(srn) {
     implicit request =>
-      val memberList = request.userAnswers.map(MembersDetailsPages(srn)).values.toList
+      val memberMap = request.userAnswers.map(MembersDetailsPages(srn))
+      val maxIndex: Int = memberMap.keys.map(_.toInt).maxOption.getOrElse(-1)
```

Comment:
is this "-1" logic working properly when MembersDetailsPages returns empty ? I mean ((0 to -1))